src/util/std_types.h

```diff
@@ -223,6 +223,16 @@ class struct_union_typet:public typet
     {
       return set(ID_C_is_padding, is_padding);
     }
+
+    bool get_is_final() const
+    {
+      return get_bool(ID_final);
+    }
+
+    void set_is_final(const bool is_final)
```

Comment:
The name makes me cry.

Reply:
So ... what would you recommend instead?

Reply:
FYI we already use `(s|g)et_is_final` for methods. So whatever we choose we should remain consistent.